src/pages/en/guides/markdown-content.md

```diff
@@ -106,6 +106,15 @@ const { frontmatter, url } = Astro.props;
   </body>
 </html>
 ```
+Be sure to have the `site` variable in your `astro.config.mjs` file.
+```
+...
+export default defineConfig({
+	...
+	site: "http://localhost:3000",
+	...
+});
+```
```

Comment:
In which case we can remove this too. Sorry! 😅 

````suggestion
Be sure to have the `site` variable in your `astro.config.mjs` file.
```
...
export default defineConfig({
	...
	site: "http://localhost:3000",
	...
});
```
````